site/styles.css

```diff
@@ -360,6 +637,15 @@ footer {
     background-color: gray;
 }
 
+@media (max-width:900px) {
+    .interaction-buttons-box {
+        padding: 10px;
+        border-width: 5px;
+        font-size: 25px;
+        border-width: 3px;
```

Comment:
You have `border-width` defined twice in this same block, you can remove the first one probably.